src/playermat/Playermat.ttslua

```diff
@@ -595,11 +604,11 @@ function getDeckAreaObjects()
 end
 
 -- draws the specified number of cards (reshuffling of discard pile is handled separately)
+---@param cardOrDeck tts__Object Card/Deck to draw from
```

Comment:
Re-using the existing reference since this is only called when the deck area objects are already determined